docs/content/getting-started/create-new-project.mdx

```diff
@@ -131,14 +131,16 @@ Inside of the `my_dagster_project/` directory, the following files and directori
       </td>
     </tr>
     <tr>
-      <td>my_dagster_project/assets/</td>
+      <td>my_dagster_project/assets.py</td>
       <td>
-        A Python package that contains all the{" "}
-        <a href="/concepts/assets/software-defined-assets">assets</a>. It is
-        helpful to include all assets in a subpackage so that you can use{" "}
-        <PyObject object="load_assets_from_package_module" /> to load assets
-        into your project, rather than needing to add assets to the
-        project every time you define one.
+        A Python module that contains {" "}
+        <a href="/concepts/assets/software-defined-assets">software-defined assets</a>.
+        <br />
+        <br />
+        Note: As your project grows, we recommend organizing assets in subpackages, e.g. <code>my_dagster_project/assets/analytics.py</code>, so that you can use{" "}
```

Comment:
`my_dagster_project/assets/analytics.py` is an example of a module not a subpackage. Subpackages are folders, so the comment is confusing.

Reply:
updated to separate the wording for package and module